### DISCUSSION TITLE
Replicating GANN regression examples.

### DISCUSSION THREAD

**Discussion by mjcallaway** [ahmedfgad/GeneticAlgorithmPython#218] [General]
Hi, I've tried running the code for the two regression examples in the documentation here:
https://pygad.readthedocs.io/en/latest/gann.html#examples

but a typical fitness value plot is not as expected, being much worse than those shown; predictions are also very poor and errors are large. Am I missing something, or have the plots in the documentation been specially selected and are not typical outputs?

**Regression Example 1 output**
![image](https://github.com/ahmedfgad/GeneticAlgorithmPython/assets/109754834/24489b96-885e-41db-9500-2938d84944fc)
Predictions of the trained network : [array([0.33516124]), array([1.38754763])]
Absolute error : 0.7.

**Regression Example 2 output**

![image](https://github.com/ahmedfgad/GeneticAlgorithmPython/assets/109754834/77b081f5-6898-4d42-b8a0-7e70e9e1296c)
Absolute error : 290.8024640876485.

**Comment by ahmedfgad** [comment]
Hi @mjcallaway,

For the first example, the code published in the documentation is not updated. Please use the code in this script: https://github.com/ahmedfgad/GeneticAlgorithmPython/blob/master/examples/gann/example_regression.py. The documentation will be updated soon.

For the second example (fish classification), I tested it and the fitness does not pass 0.0034. But maybe the result posted at the documentation is reached after making some changes in mutation or initial population. Not sure yet to be honest.

**Reply by mjcallaway** [reply]
Hi @ahmedfgad,

Thanks for your reply and pointing me to the updated script.

Also, thanks for writing this useful library :-)
